### PR TITLE
HW6 Oleksii Makarov

### PR DESCRIPTION
**Pull request by AlkMkr**

### CODE REVIEW

File: Oleksii_Makarov/HW6/Lesson6_1.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+rand_tuples = list()
+
+for temp in range(0, 100):
+    right = randint(0, 100)
+    left = randint(0, 100)
+    operation = randint(1, 3)
+    rand_tuples.append((right, left, operation))
+
+os.makedirs('test/data')
```
Comment (bohdandovbsyh) [inline Oleksii_Makarov/HW6/Lesson6_1.py]: what if I want to run your program for the second time?

File: Oleksii_Makarov/HW6/Lesson6_3.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import re
+
+with open('text.txt', 'r') as file:
+    text = file.read()
+
+letter_count = {}
+# This regex fucntion replaces almost every dash "-" with "Ђв" and I have no idea why
+# Hence telling it to also replace them. And it's working for some reason
+only_letters = re.sub(r'\d|\W|Ђ|в', '', text)
```
Comment (bohdandovbsyh) [inline Oleksii_Makarov/HW6/Lesson6_3.py]: you can try [^a-zA-Z]

File: Oleksii_Makarov/HW6/Lesson6_1.py (hunk 1 above)
Comment (AlkMkr) [reply Oleksii_Makarov/HW6/Lesson6_1.py]: > what if I want to run your program for the second time?

Added if for os.makedirs

File: Oleksii_Makarov/HW6/Lesson6_3.py (hunk 2 above)
Comment (AlkMkr) [reply Oleksii_Makarov/HW6/Lesson6_3.py]: > you can try [^a-zA-Z]

Done, it worked

File: Oleksii_Makarov/HW6/Lesson6_1.py [hunk 3]
```diff
@@ -2,14 +2,15 @@
...
     right = randint(0, 100)
     left = randint(0, 100)
     operation = randint(1, 3)
     rand_tuples.append((right, left, operation))
-
-os.makedirs('test/data')
+if os.path.exists('./test/data'):
+    pass
+else:
+    os.makedirs('./test/data')
```
Comment (bohdandovbsyh) [inline Oleksii_Makarov/HW6/Lesson6_1.py]: try to shorter this to 2 lines))

Comment (AlkMkr) [reply Oleksii_Makarov/HW6/Lesson6_1.py]: > try to shorter this to 2 lines))

shortened to one

Comment (bohdandovbsyh) [reply Oleksii_Makarov/HW6/Lesson6_1.py]: Even better)

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**